components/raftstore/src/store/util.rs

```diff
@@ -737,6 +741,62 @@ impl<
     }
 }
 
+pub struct EntryRequestRef<'a> {
```

Comment:
Please add comment explain what's the purpose of the struct.

Reply:
Done